frontend/src/assets/css/header.scss

```diff
@@ -0,0 +1,31 @@
+$navbar-bg: #f8f9fa;
```

Comment:
header.module.scss should be in header component folder, we will use scss modules for components